src/site/content/en/blog/fast-playback-with-preload/index.md

```diff
@@ -0,0 +1,604 @@
+      <td rowspan=2 style="white-space: nowrap">
+<a href="#link_preload">Link preload</a>
+      </td>
+      <td>
+Forces the browser to make a request for a video resource without blocking
+the document's <code>onload</code> event.
+      </td>
+      <td>
+HTTP Range requests are not compatible.
+      </td>
+    <tr>
+      <td>
+Compatible with MSE and file segments.
+      </td>
+      <td>
+Should be used only for small media files (&lt;5 MB) when fetching full resources.
+      </td>
+    </tr>
+    <tr>
+      <td>
+<a href="#manual_buffering">Manual buffering</a>
+      </td>
+      <td>
+Full control
+      </td>
+      <td>
+Complex error handling is the website's responsibility.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Video preload attribute
+
+If the video source is a unique file hosted on a web server, you may want to
+use the video `preload` attribute to provide a hint to the browser as to [how
+much information or content to preload][preload]. This means [Media Source Extensions
+(MSE)][mse-basics] is not compatible with `preload`.
+
+Resource fetching will start only when the initial HTML document has been
+completely loaded and parsed (e.g. the `DOMContentLoaded` event has fired)
+while the very different `window.onload` event will be fired when resource
+has actually been fetched.
+
+<figure>
+  <img src="./video-preload.svg">
+</figure>
+
+Setting the `preload` attribute to `metadata` indicates that the user is not
+expected to need the video, but that fetching its metadata (dimensions, track
+list, duration, and so on) is desirable. Note that starting in [Chrome
+64][chrome64], the default value for `preload` is `metadata`. (It was `auto`
+previously).
+
+
+```js
+<video id="video" preload="metadata" src="file.mp4" controls></video>
+
+<script>
+  video.addEventListener('loadedmetadata', function() {
+    if (video.buffered.length === 0) return;
+
+    const bufferedSeconds = video.buffered.end(0) - video.buffered.start(0);
+    console.log(`${bufferedSeconds} seconds of video are ready to play.`);
+  });
+</script>
+```
+
+Setting the `preload` attribute to `auto` indicates that the browser may cache
+enough data that complete playback is possible without requiring a stop for
+further buffering.
+
+```js
+<video id="video" preload="auto" src="file.mp4" controls></video>
+
+<script>
+  video.addEventListener('loadedmetadata', function() {
+    if (video.buffered.length === 0) return;
+
+    const bufferedSeconds = video.buffered.end(0) - video.buffered.start(0);
+    console.log(`${bufferedSeconds} seconds of video are ready to play.`);
+  });
+</script>
+```
+
+There are some caveats though. As this is just a hint, the browser may completely
+ignore the `preload` attribute. At the time of writing, here are some rules
+applied in Chrome:
+
+- When [Data Saver][datasaver] is enabled, Chrome forces the `preload` value to
+  `none`.
+- On a cellular connection (2G, 3G, and 4G), Chrome forces the `preload` value to
```

Comment:
Please restore "- In Android 4.3, Chrome forces the `preload` value to `none` due to an [Android bug]." as communicated earlier at https://github.com/GoogleChrome/web.dev/pull/4064#discussion_r507831887

Reply:
@jpmedley This is **valuable** information for web developers that still want to know why Chrome on Android 4.3 (not supported anymore due to [JellyBean deprecation](https://bugs.chromium.org/p/chromium/issues/detail?id=923900)) overrides the `preload` value.
This bug is still relevant. **PLEASE** restore.

Reply:
@jpmedley (gentle ping)